content/applications/inventory_and_mrp/manufacturing/management/work_center_time_off.rst

```diff
@@ -0,0 +1,121 @@
+so ensures that manufacturing operations can continue until the impacted work center is available
+again.
+
+Configuration
+=============
+
+Before a work center can be designated as unavailable, the Odoo platform must be properly
+configured. First, it is necessary to enable developer mode. This allows the :guilabel:`Time Off`
+smart button to appear on each work center's :guilabel:`Working Hours` pop-up window.
+
+Enable developer mode by going to :guilabel:`Settings`, scrolling to the bottom of the page, and
+clicking :guilabel:`Activate the developer mode` under the **Developer Tools** heading.
+
+.. image:: work_center_time_off/developer-mode-button.png
+   :align: center
+   :alt: The "Activate the developer mode" button.
+
+Next, install the :guilabel:`Time Off` app. This is the app used for assigning time off to all
+resources within Odoo, including employees and work centers. Navigate to :menuselection:`Apps`, then
+type "Time Off" in the :guilabel:`Search...` bar. The card for the :guilabel:`Time Off` module
+should be the only one that appears on the page. Click the green :guilabel:`Install` button on the
+card to install the app.
+
+.. image:: work_center_time_off/time-off-install-card.png
+   :align: center
+   :alt: The Time Off module installation card.
+
+The last step is to properly configure work centers. For this workflow, it is necessary to have at
+least two work centers: one that is made unavailable and a second that receives the work orders that
+the other cannot accept. If no second work center is configured, Odoo cannot route work orders away
+from the unavailable work center and they will pile up in its queue.
+
+To create a work center, navigate to :menuselection:`Manufacturing --> Configuration --> Work
+Centers --> Create`.
+
+Make sure that both work centers have the same equipment listed under the :guilabel:`Equipment` tab.
+This ensures that operations carried out at one work center can also be performed at the other.
+
+.. image:: work_center_time_off/work-center-equipment-tab.png
+   :align: center
+   :alt: The equipment tab on a work center form.
+
+For the work center that will be made unavailable, select the second work center on the
+:guilabel:`Alternative Workcenters` drop-down menu. Now, Odoo knows to send work orders to the
+second work center when the first is unavailable for any reason.
+
+.. image:: work_center_time_off/alternative-work-center-selection.png
+   :align: center
+   :alt: A work center form configured with an alternative work center.
+
+Add time off for a work center
+==============================
+
+With configuration completed, time off can now be assigned to the work center that will be made
+unavailable. Begin by navigating to :menuselection:`Configuration --> Work Centers` and selecting
+the affected work center. Click :guilabel:`Edit`, and then the :guilabel:`External link` button next
+to the :guilabel:`Working Hours` drop-down menu.
```

Comment:
I think it would be good to reiterate that we're in the Manufacturing app, and the external link could use an icon here or just lowercase the label since the text does no appear on screen
```suggestion
unavailable. Begin by navigating to :menuselection:`Manufacturing --> Configuration --> Work
Centers` and selecting the affected work center. Click :guilabel:`Edit`, and then the :guilabel:`↗️
(external link)` button next to the :guilabel:`Working Hours` drop-down menu.
```
OR 
```suggestion
unavailable. Begin by navigating to :menuselection:`Manufacturing --> Configuration --> Work
Centers` and selecting the affected work center. Click :guilabel:`Edit`, and then the
:guilabel:`external link` button next to the :guilabel:`Working Hours` drop-down menu.
```